### ISSUE TITLE
MIR-borrowck: diagnostic says "use" (as noun?) rather than "used" (verb, past tense)

### ISSUE THREAD

**Issue by pnkfelix** [rust-lang/rust#45960]
#45877 added a [ui](https://github.com/rust-lang/rust/blob/3acb4e9e4a30cd5c413b094a0821426951d0745c/src/test/ui/borrowck/borrowck-reinit.stderr) test that shows that the AST-borrowck and MIR-borrowck are *very* close in their diagnostic... so close that its easy to overlook the following change to the note output:

Namely, the file `src/test/ui/borrowck/borrowck-reinit.stderr` says,
for AST:
```
   |                ^ value used here after move 
```

but for MIR:
```
   |                ^ value use here after move 
```

Nota bene, fixing this is not a trivial one liner, because the function that constructs the error diagnostic is parameterized over a string describing the action (i.e. a verb like "use" or "borrow"). Still, it should be a relatively easy thing to resolve.

**Comment by kennytm** [comment]
The "verb" (`desired_action`) is passed in when calling `check_if_path_is_moved` in https://github.com/rust-lang/rust/blob/master/src/librustc_mir/borrow_check.rs, which currently includes:

* "update"
* "borrow"
* "use"
* "assignment" (?!)

The only code consuming the `desired_action` in an error message is: 

* `format!("value {} here after move", desired_action)`

So I think this is really just a trivial 5-liner to change the tense of all of these.

**Comment by nikomatsakis** [comment]
Some context about what we are trying to do in the NLL / Borrow Checker etc is available in [this paper document](https://paper.dropbox.com/doc/Non-Lexical-Lifetimes-rustc-Dev-Practices-cdPdAj0eXGt5H4FPWGRve). The other thing is that I wrote [this gist that talks about how to update a test to work for both AST and MIR borrowck](https://gist.github.com/nikomatsakis/a0d5dee0bd3c4d4f0442914a9abdf74c), though in this case it may not be needed.

**Comment by austinbes** [comment]
I'd like to give this one a shot

**Comment by ritiek** [comment]
This issue can be closed as of #46231.

**Comment by kennytm** [comment]
Closing as fixed by #46231.
